lib/python/setup.cfg

```diff
@@ -0,0 +1,23 @@
+[metadata]
+name = didkit
+version = 0.1.0
+author = Gregório Granado Magalhães
```

Comment:
can pls change to
Spruce Systems, Inc.
oss@spruceid.com

If people want to see who wrote what, they can look up the commit history. Thanks!

Reply:
Whoops, forgot to change actually